cobc/cobc.c

```diff
@@ -6533,16 +6481,103 @@ line_has_listing_statement (char *line, const enum cb_format source_format)
 	return 1;
 }
 
+static char *
+get_word (char *str)
+{
+	int i = 0;
+
+	if (!str) {
+		return NULL;
+	}
+	while (str[i] && isspace (str[i])) {
```

Comment:
What about separator commas and separator semicolons?
The old code used `isspace ((unsigned char)*bp) || *bp == ',' || *bp == '.' || *bp == ';'`.

Reply:
I also put that before but when I read the documentation I didn't understand why these characters is understood as a space and this not change the result of the test. I'll try other test for compare old replacing with the new replacing for check the differences

Reply:
Note: it obviously would be good to have this in the testsuite, too.
In effect the point is that there are multiple "separators" - all "space characters", newline, separator semicolon and separator comma.
The last one is a bit tricky as, depending on `DECIMAL-POINT IS COMMA` this may or may not mean that "3,2" has to be separated (=2 "words"/tokens), while "3,a" would always mean two tokens. [note: the listing may "don't care" about that at all or always ignore the comma if `DECIMAL-POINT IS COMMA` is in effect].

Reply:
I _think_ those testcases are missing, right?